src/main/java/com/cleanroommc/modularui/screen/ModularContainer.java

```diff
@@ -46,6 +46,7 @@ public static ModularContainer getCurrent(EntityPlayer player) {
     private final List<ModularSlot> slots = new ArrayList<>();
     private final List<ModularSlot> shiftClickSlots = new ArrayList<>();
     private ContainerCustomizer containerCustomizer;
+    private ModularScreen modularScreen = null;
```

Comment:
This looks like it can cause a crash on server side.

Reply:
how exactly? JEI's `transferRecipe()` is only called on client side, and if it was called server side it would fail the `instanceof` check

Reply:
nvm i think i see what you mean